lib/src/data/data_manager.dart

```diff
@@ -1108,12 +1110,7 @@ class DataManager {
           stackTrace: stacktrace,
         );
       }
-    } else {
-      log('\tLayout [$layoutID] has no conditions.');
-    }
 
-    if (model.updates.variables.containsKey(layoutID)) {
-      log('\tLayout [$layoutID] has variables.');
```

Comment:
With your removal of this if statement, you're now checking `model.updates.conditions.containsKey(layoutID)`, and then processing variables regardless, without checking for variables separately. 

The conditions check is defining the behavior of the variables getOrFetch function.